### PR TITLE
Create component info file for building releases

### PR DESCRIPTION
**Pull request by rubencatshoek**
I created the `component_info` file that is required for buildscript. The follow example was used: https://github.com/OpenConext/Stepup-SelfService/blob/develop/component_info

### CODE REVIEW

**Comment by quartje** [comment]
I think this should work. Let's merge and find out!

**Review by quartje — APPROVED** [review]

File: component_info [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PHP_VERSION=82
+SYMFONY_VERSION=6
+ENCORE=yes
```
Comment (MKodde) [inline component_info:3]: Webpack Encore wil probably not be used

File: component_info [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+NODE_VERSION=16
+ASSETIC=no
+COMPOSER_VERSION=2
```
Comment (MKodde) [inline component_info:6]: Please add a `crlf` to the end of the file

**Review by MKodde — APPROVED** [review]
One nitpick, one more important point :arrow_double_down:

**Pull request merged**